rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/Util.rsc

```diff
@@ -129,3 +130,9 @@ bool(&T, &T) desc(bool(&T, &T) f) {
         return f(t2, t1);
     };
 }
+
+set[&T] flatMap(set[&S] ss, set[&T](&S) f) =
```

Comment:
might be more compact to write as a reducer, as then we can use set concat?

```rascal
= ({} | it + f(s) | s <- ss);
```

(internally, there is __some__ performance gain of writing a single set comprehension, but I would only do that if it's in the profiles.)